pkg/collector/process.go

```diff
@@ -46,6 +46,8 @@ type collectingProcess struct {
 	messages []*entities.Message
 	// maps each client to its client handler (required channels)
```

Comment:
Not related to this PR. I feel it is better to have uniformity/convention when adding comments to the fields, i.e., starting the comment with the field name. what do you say?

Reply:
Makes sense. I will create a PR to uniform all the comments. Thanks!